datafusion/sqllogictest/test_files/unnest.slt

```diff
@@ -524,10 +533,19 @@ select unnest(column1) from (select * from (values([1,2,3]), ([4,5,6])) limit 1
 5
 6
 
-## FIXME: https://github.com/apache/datafusion/issues/11198
```

Comment:
This issue has been closed. The error assertion makes sense to me. I also added another positive test at L539